src/Uno.UI.Tests/Windows_UI_Input/Given_GestureRecognizer.cs

```diff
@@ -1003,7 +1017,7 @@ internal static class GestureRecognizerTestExtensions
 			var frameId = (uint)Interlocked.Increment(ref _frameId);
 			id = id ?? 1;
 			ts = ts ?? frameId;
-			var pointer = new PointerDevice(device ?? PointerDeviceType.Mouse);
+			var pointer = new PointerDevice(device ?? PointerDeviceType.Touch);
```

Comment:
Why do you change this default?

Reply:
Because the translate thresold for mouse is now == 1 like UWP. It's easier to change the pointer type for all tests at once than update all tests that were assuming that 1 pixel would be lower than the threashold.